library/general/src/modules/PackagesProposal.rb

```diff
@@ -190,6 +193,71 @@ def RemoveResolvables(unique_id, type, resolvables, optional: false)
       true
     end
 
+    # Adds a list of resolvables to do not be installed
+    #
+    # @param [String] unique_id the unique identificator
+    # @param [Array<String>] resolvables list of resolvables to taboo
+    # @return [Boolean] whether successful
+    def AddTaboos(unique_id, resolvables)
+      return false if !CheckParams(unique_id, :package)
```

Comment:
To be honest, I do not get logic here. it should always succeed for :package...and it just check that unique id is not nil or empty. Maybe second parameter no longer makes sense?